index.html

```diff
@@ -9,18 +9,35 @@
         <script src="js/fruits_and_veggies_test.js"></script>
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }